tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we have an option to avoid additional file in the project root?

Reply:
I think not — TS doesn't have the ability to store configs in package.json. Docs: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html

Reply:
You can specify options for TS directly in the jest config (in `package.json`), but this is a bad practice — other toolses will not be able to use this config.

More: https://kulshekhar.github.io/ts-jest/user/config/tsConfig#inline-compiler-options

Reply:
OK, let’s keep the extra config.

Reply:
If the TS takes root, the config can be moved to [`sharec-config`](https://github.com/logux/sharec-config).